api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js

```diff
@@ -5,6 +5,30 @@ angular.module('vetDetails')
         var self = this;
         //var vetId = $stateParams.vetId || 0;
 
+        this.show = ($event, vetID) => {
+            let child = document.getElementsByClassName("m" + vetID)[0];
+            let left = $event.pageX;
+            let top = $event.clientY;
+            if (document.documentElement.clientWidth > 960) {
+                child.style.left = (left + 221) + 'px';
+            }
+            if (document.documentElement.clientWidth < 420) {
+                child.style.left = (170) + 'px';
+            } else if (document.documentElement.clientWidth < 510) {
+                child.style.left = (left + 334.5 / 2.5) + 'px';
+            } else {
+                child.style.left = (left + 200) + 'px';
+            }
+            child.style.top = (top) + 'px';
+            child.classList.remove("modalOff");
+            child.classList.add("modalOn");
+        }
```

Comment:
Why are you styling using JS rather than CSS

Reply:
Since it's similar to an onHover event, I'm making the div that contains my badge information display based on the user's mouse position when they hover over the badge.